tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/StorageManager.java

```diff
@@ -538,7 +538,8 @@ public Appender getAppender(OverridableConf queryContext,
     if (scannerClass == null) {
       scannerClass = conf.getClass(
           String.format("tajo.storage.scanner-handler.%s.class", handlerName), null, Scanner.class);
-      SCANNER_HANDLER_CACHE.put(handlerName, scannerClass);
+      if (scannerClass != null)
```

Comment:
Hi @dongjoon-hyun 

Thank you for your detailed comments.
I just leaved the comment because I couldn't create new table without reference to the IF statement as following:

```
default> create table test1 (id int) using orc;
ERROR: Store format orc is not supported.
```

Anyway, I can't reproduce the NPE now because of above situation. I think that it may be related to other source codes. If possible, could you check it again?